openedx/features/discussions_plugin_piazza/app.py

```diff
@@ -0,0 +1,25 @@
+from lms.djangoapps.discussiondjango_comment_client.utils import is_discussion_enabled
```

Comment:
This import is invalid, it is likely why this plugin won't load.

Reply:
Removed entirely in next commit